src/OrleansVSTools/VSProjectTemplateGrainInterfaceVB/VSProjectTemplateGrainInterfaceVB.vbproj

```diff
@@ -64,6 +64,11 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="My Project\AssemblyInfo.vb" />
+    <Compile Include="Settings1.Designer.vb">
+      <AutoGen>True</AutoGen>
+      <DesignTimeSharedInput>True</DesignTimeSharedInput>
+      <DependentUpon>Settings.settings</DependentUpon>
+    </Compile>
```

Comment:
hmmm, I don't think it's related to NuGet. It just added a new designer for Settings.settings (although there was already one). I think we don't need a Settings.settings in the first place (unless it's mandatory for VB, but I don't think so).
Anyway, I'll merge and we can clean up separately in a new PR, as you mentioned